lib/swarm/tracker/tracker.ex

```diff
@@ -120,9 +120,13 @@ defmodule Swarm.Tracker do
     # wait for node list to populate
     nodelist = Enum.reject(Node.list(:connected), &ignore_node?/1)
 
-    strategy = Node.self
-           |> Strategy.create()
-           |> Strategy.add_nodes(nodelist)
+    strategy =
+      case ignore_node?(Node.self) do
```

Comment:
Hi @bitwalker , do we have any updates about it? Please kindly let me know if anything I can help, thanks for your time.